nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -198,6 +200,9 @@ def process_all_tracks(df, dataset, remove_growth_outliers, num_workers):
     df = add_features.add_non_interphase_size_shape_flag(df)
     df = add_change_over_time(df)
     df = add_neighborhood_avg_features.run_script(df, num_workers=num_workers)
+    df = add_neighborhood_avg_features_lrm.run_script(df, num_workers=num_workers, 
```

Comment:
exclude outliers is false? I think exclude outliers needs to be set to true so that, minimally, bad segmentations are filtered using  `df = all_timepoints_minimal_filtering(df)`.

Reply:
Great catch thank you!